### PR TITLE
diff printing fix

### PR DESCRIPTION
**Pull request by bheni**
Columns with % signs were being printed using printf style functions which were looking at %s and escape characters.

### CODE REVIEW

**Review by zachmu — APPROVED** [review]
bats test?

**Comment by timsehn** [comment]
The bats test for this last time was pretty clever. Having a hard time locating it.

**Comment by bheni** [comment]
@tim found the test:

```go
@test "validate string formatting" {
      dolt sql <<SQL
CREATE TABLE test2 (
  str varchar(256) NOT NULL,
  PRIMARY KEY (str)
);
SQL
  TESTSTR='0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ`~!@#$%^&*()){}[]/=?+|,.<>;:_-_%d%s%f'
  dolt sql -q "INSERT INTO test2 (str) VALUES ('$TESTSTR')"

  run dolt sql -q "SELECT * FROM test2"
  [ $status -eq 0 ]
  [[ "$output" =~ "$TESTSTR" ]] || false

  run dolt sql -q "SELECT * FROM test2" -r csv
  [ $status -eq 0 ]
  [[ "$output" =~ "$TESTSTR" ]] || false

  run dolt sql -q "SELECT * FROM test2" -r json
  [ $status -eq 0 ]
  [[ "$output" =~ "$TESTSTR" ]] || false
}
```

**Comment by timsehn** [comment]
`dolt diff` at the end and we good. I was grepping for MISSING in the bats
directory.

File: bats/sql-shell.bats [hunk 1]
```diff
@@ -77,4 +80,12 @@ SQL
   run dolt sql -q "SELECT * FROM test2" -r json
   [ $status -eq 0 ]
   [[ "$output" =~ "$TESTSTR" ]] || false
+
+  dolt add .
```
Comment (timsehn) [inline bats/sql-shell.bats:84]: Why not just diff the working set?

**Review by timsehn — APPROVED** [review]

**Comment by timsehn** [comment]
This fixes: #1389

**Pull request merged**